LocalExtensions.php

```diff
@@ -194,6 +194,22 @@
 
 if ( $wmgUseCommentStreams ) {
 	wfLoadExtension ( 'CommentStreams' );
+	$wi->config->settings['wgManageWikiNamespacesAdditional']['default']['wgCommentStreamsAllowedNamespaces'] = [
+		'name' => 'Can comments appear in this namespace?',
+		'main' => true,
+		'talk' => true,
+		'blacklisted' => [],
+		'vestyle' => false,
+		'overridedefault' => -1,
+	];
+	$wi->config->settings['wgManageWikiNamespacesAdditional']['default']['wgCommentStreamsEnableTalk'] = [
```

Comment:
Actually, looking at https://www.mediawiki.org/wiki/Extension:CommentStreams

This is a boolean. So should be in MW/LS.

Reply:
hum, when moved it will follow the format of  wgCommentStreamsNewestStreamsOnTop ?